src/screens/PoolTogether/PoolTogetherDashboard.js

```diff
@@ -42,16 +42,16 @@ import CircleButton from 'components/CircleButton';
 import ActivityFeed from 'components/ActivityFeed';
 import RetryGraphQueryBox from 'components/RetryGraphQueryBox';
 
-// models
+// types
 import type { Accounts } from 'models/Account';
-import type { Balances } from 'models/Asset';
 import type { PoolPrizeInfo } from 'models/PoolTogether';
 import type { Dispatch, RootReducerState } from 'reducers/rootReducer';
```

Comment:
This one stands about a bit but i'm starting to realise that we have a mix of type definitions in models / reducers - just a note for something we could sort out later.